libs/shared/fluid-utils/src/lib/group-navigation.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ARROW_LEFT, ARROW_RIGHT } from '@dynatrace/shared/keycodes';
+
+/** Calculate based on a length and the direction (arrow key) what the next index is supporting rotationing */
+export function getNextIndex(
```

Comment:
please provide a more meaningful name that reflects what the function does.

Maybe `getTabGroupNavigationIndex`. (please name the file always in a kebab-cased variant of the method so it is easier to find)

Please update the jsdoc to reflect the problem more took me a while to understand what it is doing

- [ ] Please add a simple unit test for it :) Pure functions are easy to test

Reply:
it is not only for tab groups, but for grouped items that you can navigate with the arrow keys in general, right?
so maybe something like `getNextGroupItemIndex`?